src/devices/src/virtio/net/device.rs

```diff
@@ -851,10 +867,8 @@ pub(crate) mod tests {
 
         fn set_mac(&mut self, mac: MacAddr) {
             self.guest_mac = Some(mac);
-            let mut config_space;
-            config_space = vec![0; MAC_ADDR_LEN];
-            config_space[..].copy_from_slice(mac.get_bytes());
-            self.config_space = config_space;
+            self.config_space = ConfigSpace::default();
```

Comment:
this line should be removed:
- right now it's just superfluous since `guest_mac` is overwritten by the next instruction anyway.
- in a future where `ConfigSpace` contains more than just a `guest_mac`, this line would actually be a **bug** since it would change other fields of config space on a mac setter function.

Reply:
That's right! Removed it.